### PR TITLE
Create a Gem - Fixed the o3de command examples. The previous version assumed users h…

### PR DESCRIPTION
**Pull request by micronAMZN**
…ave <engine>/scripts in their system path which is not consistent with any other docs.

Signed-off-by: Mike Cronin <58789750+micronAMZN@users.noreply.github.com>

### CODE REVIEW

File: content/docs/user-guide/gems/development/creating/_index.md [hunk 1]
```diff
@@ -8,10 +8,10 @@ You can develop your own independent modules to distribute and build as Open 3D
 bundle of code and/or assets, a manifest file, a CMake build file, and an optional display icon for project configuration
 tools. This is all that's needed to ship a Gem.
 
-Although you can create a Gem manually by creating all of the files yourself, it's recommended to use the `o3de` tool:
+Although you can create a Gem manually by creating all of the files yourself, it's recommended to use the `o3de` tool located in the `<engine>/scripts` directory:
 
 ```cmd
-o3de create-gem -gp <path to create gem at>
+o3de.bat create-gem -gp <path to create gem at>
```
Comment (willihay) [inline content/docs/user-guide/gems/development/creating/_index.md:14]: Ideally we should be showing both Windows and Linux examples now using the `tabs` shortcode. (See `user-guide/project-config/cli-reference.md` for an example of its use with o3de.bat/o3de.sh.)

**Review by willihay — CHANGES_REQUESTED** [review]

File: content/docs/user-guide/gems/development/creating/_index.md (hunk 1 above)
Comment (micronAMZN) [reply content/docs/user-guide/gems/development/creating/_index.md]: Done

File: content/docs/user-guide/gems/development/creating/_index.md [hunk 2]
```diff
@@ -8,12 +8,25 @@ You can develop your own independent modules to distribute and build as Open 3D
...
 
-Although you can create a Gem manually by creating all of the files yourself, it's recommended to use the `o3de` tool:
+Although you can create a Gem manually by creating all of the files yourself, it's recommended to use the `o3de` tool located in the `<engine>/scripts` directory:
+
+{{< tabs name="Create a Gem" >}}
+{{% tab name="Windows" %}}
 
 ```cmd
-o3de create-gem -gp <path to create gem at>
+<engine>/scripts/o3de.bat create-gem -gp <path to create gem at>
```
Comment (willihay) [inline content/docs/user-guide/gems/development/creating/_index.md:17]: Using a forward slash will fail in the Windows command prompt. Prefer back slash for Windows-specific command examples.
```suggestion
<engine>\scripts\o3de.bat create-gem -gp <path to create gem at>
```

File: content/docs/user-guide/gems/development/creating/_index.md [hunk 3]
```diff
@@ -50,14 +63,29 @@ Your `CMakeLists.txt` file is like any other CMake file. When you create it, kee
...
 
-When you create a Gem without a specified template by using the command `o3de create-gem -gp <path to create a gem at>`, it's created from the "DefaultGem" template. This creates a Gem for packaging both code and assets. You can also create a Gem that provides only assets by specifying the "AssetGem" template through the `create-gem` command:
+When you create a Gem without a specified template, it's created from the "DefaultGem" template. This creates a Gem for packaging both code and assets. You can also create a Gem that provides only assets by specifying the "AssetGem" template through the `create-gem` command:
+
+{{< tabs name="Create an asset Gem" >}}
+{{% tab name="Windows" %}}
 
 ```cmd
-o3de create-gem --gem-path <path to create gem at> --template-name AssetGem
+<engine>/scripts/o3de.bat create-gem --gem-path <path to create gem at> --template-name AssetGem
```
Comment (willihay) [inline content/docs/user-guide/gems/development/creating/_index.md:72]:
```suggestion
<engine>\scripts\o3de.bat create-gem --gem-path <path to create gem at> --template-name AssetGem
```

File: content/docs/user-guide/gems/development/creating/_index.md [hunk 4]
```diff
@@ -50,14 +63,29 @@ Your `CMakeLists.txt` file is like any other CMake file. When you create it, kee
...
 # or
-o3de create-gem --gem-path <path to create gem at> --template-path <engine-root>\Templates\AssetGem
+<engine>/scripts/o3de.bat create-gem --gem-path <path to create gem at> --template-path <engine-root>\Templates\AssetGem
```
Comment (willihay) [inline content/docs/user-guide/gems/development/creating/_index.md:74]:
```suggestion
<engine>\scripts\o3de.bat create-gem --gem-path <path to create gem at> --template-path <engine-root>\Templates\AssetGem
```

File: content/docs/user-guide/gems/development/creating/_index.md [hunk 5]
```diff
@@ -50,14 +63,29 @@ Your `CMakeLists.txt` file is like any other CMake file. When you create it, kee
...
 ```
 
+{{% /tab %}}
+{{% tab name="Linux" %}}
+
+```cmd
+<engine>/scripts/o3de.sh create-gem --gem-path <path to create gem at> --template-name AssetGem
+# or
+<engine>/scripts/o3de.sh create-gem --gem-path <path to create gem at> --template-path <engine-root>\Templates\AssetGem
```
Comment (willihay) [inline content/docs/user-guide/gems/development/creating/_index.md:83]: I bet Linux will need paths in parameters to be forward slash too.
```suggestion
<engine>/scripts/o3de.sh create-gem --gem-path <path to create gem at> --template-path <engine-root>/Templates/AssetGem
```

**Review by willihay — CHANGES_REQUESTED** [review]

**Review by willihay — APPROVED** [review]

**Review by chanmosq — APPROVED** [review]

**Pull request merged**